tests/docker/conftest.py

```diff
@@ -0,0 +1,72 @@
+from shutil import which
```

Comment:
These fixtures ensure that everything (Docker images and containers) is cleaned up when the test suite ends.